### DISCUSSION TITLE
WOL on windows

### DISCUSSION THREAD

**Discussion by VoooDooCheese** [AlexGustafsson/homebridge-wol#134] [Questions] [answered]
Hey,
I am having some trouble getting the WOL to work,

I have the shutdown command working which is awesome.
But i cant work out what command to use for the Wake? (im using pi homebridge, to wake a win 10pc)

**Comment by AlexGustafsson** [comment, marked as answer]
See the section in the README about turning the device on (see below).

You should at the very least have the `mac` key specified to your Windows PC's MAC address to use Wake On LAN.

The `startCommand` and `wakeCommand` are quite special and not applicable in every case. They're only used when one requires more customization.

#### Turning on

| Key | Description |
| --- | ------------|
| mac | The device's MAC address - used to send Magic Packets. Allows any format such as `XX:XX:XX:XX:XX:XX` or `XXXXXXXXXXXX` |
| broadcastAddress | The broadcast address to use when sending the Wake on LAN packet |
| startCommand | Command to run in order to start the machine |
| startCommandTimeout | Timeout for the start command in seconds. Use 0 (default) to disable the timeout |
| wakeGraceTime | Number of seconds to wait after startup before checking online status and issuing the `wakeCommand`, default `45` |
| wakeCommand | Command to run after initial startup, useful for macOS users in need of running `caffeinate` |
| wakeCommandTimeout | Timeout for the wake command in seconds. Use 0 (default) to disable the timeout |

**Reply by VoooDooCheese** [reply]
my question is more what do i write in this box? Or am i missing something and not need to put anything in there and it should be working which would lead to other problems.

![image](https://user-images.githubusercontent.com/77004351/103697343-b4fbf800-4f97-11eb-8d6b-84f29a0417a7.png)

**Reply by AlexGustafsson** [reply]
You don’t need to write anything there.

**Reply by AlexGustafsson** [reply]
Did you end up solving the issue?

**Reply by fender7083** [reply]
I have the exact same question, sorry for spamming this forum. You are saying it should work leaving that blank? I can shut down but not start

**Reply by AlexGustafsson** [reply]
The start command is only used if you have a special command to run in order to start your computer up. Examples are when you want to use this plugin to control virtual machines and there's a command such as `vm-management --start my-vm`. This is used __instead__ of the Wake On LAN capabilities of the plugin. So yes, if you intend to wake your computer up using Wake On LAN you can leave the start command blank and make sure you've set the `mac` property of the accessory.
